docs/modules/ROOT/pages/how-tos/testing/integration/debugging.adoc

```diff
@@ -0,0 +1,44 @@
+= Debugging Integration Tests
+
+In the event of an integration test scenario failing, there are a number of steps to take in order to debug the issue before rerunning the test in question. These steps are outlined below.
+
+
+.Prerequisites
+- You have a snapshot that has completed all its initial tests. All initial tests must be finished before attempting to debug the failed test.
+
+- You have access to the console in order to view the logs of the failed test.
+
+.Procedure
+
+. Identify the IntegrationTestScenario that has failed. This can be found in the console by navigating to 
+Applications/your_application_name/activity and select "Pipeline Runs" under "Activity By". Your view should be similar to the below image.
+
++
+image::activityview.png[role="border" alt="Activity View"]
+
+. Click on the pipeline run that has a status of "Failed". This will take you to a detailed view of the failed run. On the right side of this page, look for the "Log Snippet" section. This will contain the most recent error returned by the integration test. For more comprehensive logs, click on "See Logs". See the below image for reference.
+
++
+image::testfailed.png[role="border" alt="Test Failed"]
+
+. In the above example the output suggests that the pipelineRun is missing an expected parameter, "username". To edit the integration test, scroll down to the "Integration Test" field and click on the listed test. This should take you to a view similar to the one seen below.
```

Comment:
Are there different types of failure mechanisms? Are you only trying to show users how to debug and resolve missing parameter issues?

I feel like this needs to be more generic.

Reply:
The intention was to provide a step-by-step resolution of an example issue but is it the case where broader is better here? I could instead point to the editing page for clarity or else make clearer that this is just one example of an issue and how to resolve it?